tests/test_spinbox.py

```diff
@@ -0,0 +1,25 @@
+"""
+ Pylibui test suite.
+
+"""
+
+from pylibui.controls import Spinbox
+from tests.utils import WindowTestCase
+
+
+class SpinboxTest(WindowTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.spinbox = Spinbox(0, 100)
+
+    def test_value_attribute(self):
+        """Tests the spinbox's `value` attribute."""
+        self.assertEqual(self.spinbox.getValue(), 0)
```

Comment:
Same as for slider, initial value of spinbox should be zero is a possible test..